docs/building/building-archlinux-template.md

```diff
@@ -245,7 +246,7 @@ $ make meta-packages-vm
 ___
 ### 6. Build the actual Arch Linux template
 ```console
-$ make template
+# make template
```

Comment:
Hi !
The guide uses the `$` sign to indicate a command prompt ; replacing `$` with `#` would indicate a comment - which I guess isn't the case (I haven't tried building arch templates though). Can you clarify ?
Also, your commit's comment mentions "make install requires root" but I don't see anything related to running one of the commands as root; perhaps you meant `$ sudo make template` ?

Reply:
'sudo su' in a console would change the prompt to a '#', so from the context it seems like that's the intent. However, ordinarily it shouldn't be necessary to run build as root. @NebulaBC , I have seen similar behavior when I accidentally added GPG keys as root for example, then attempted to do builds. Are you sure every step prior was done at the regular user level?

Reply:
awokd: I have tried multiple times making sure to run everything as the user but it would not generate the install-templates.sh unless I ran as root.

taradiddles: Sorry, the `make install` was just a typo. Also the # does not indicate a comment, this is shown in other parts of the guide as root, and is standard practice for showing a root prompt.

Reply:
Oh, I had never paid attention that the prompt changed from `$` to `#` -  despite being a "command line" guy and using linux since pre 2000's (ouch - time is flying). I stand corrected :smile: !

As for running `make template` as root vs. as a user: I'm not a arch user but when time allows I'll try to follow the guide ; meanwhile, maybe you could run `make template` once as root and once as a regular user and `diff` the output ?

Reply:
I see someone else had the same issue here: https://github.com/Qubes-Community/Contents/pull/182#issuecomment-1096765324. When I merged that pull, it made this one conflict due to the date change. Could you please resubmit this as a new pull against the newer https://github.com/Qubes-Community/Contents/blob/master/docs/building/building-archlinux-template.md, but spell out `sudo make template` instead of only changing the command prompt to help avoid confusion?